src/table/settings/economy_settings.ini

```diff
@@ -300,3 +300,16 @@ str      = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE
 strhelp  = STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE_HELPTEXT
 post_cb  = [](auto) { InvalidateWindowClassesData(WC_COMPANY_INFRASTRUCTURE); }
 cat      = SC_BASIC
+
+[SDT_VAR]
+var      = economy.cargo_scale
+type     = SLE_UINT16
+flags    = SF_NO_NETWORK
+def      = 100
+min      = 15
```

Comment:
Why 15? Not 5? 1? :D

Reply:
We need to have at least one industry production tick per economy month/minute, and inversely scale the production callback frequency by the cargo scale percentage. The lowest this setting can go without the production interval exceeding the length of a economy month/minute is 13...which seems even more arbitrary than 15. I was thinking of making it 25, but why restrict it? Thoughts?

Reply:
This should be documented, in which case it might be better as a constant somewhere else.